src/Frontend/Themes/Fork/Core/Layout/Sass/components/_navbar.scss

```diff
@@ -0,0 +1,28 @@
+.navbar-default {
+  border: 0;
+  margin-bottom: 0;
+
+  .navbar-nav {
+
+    li {
+
+      a {
+        font-size: $font-size-small;
+        font-weight: 500;
+        -webkit-font-smoothing: antialiased;
+      }
+    }
+  }
+}
+
+.navbar-brand {
+  font-family: 'Courgette';
```

Comment:
Will this provide a decent fallback if the Courgette font doesn't load? Can you check if you need to provide a font-stack?